content/kubermatic/main/architecture/concept/kkp-concepts/applications/default-applications-catalog/_index.en.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Default Applications Catalog"
+linkTitle = "Default Applications Catalog"
+date = 2024-01-16T12:57:00+02:00
+weight = 3
```

Comment:
Sorry, another "should be set on all pages metadata" nit: Since this is an EE feature, please add

```
enterprise = true
```

to all pages :)

Reply:
Added!